packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,243 @@
+
+  // body content:
+  --token-side-nav-body-list-margin-vertical: 24px;
+  --token-side-nav-body-list-item-padding-horizontal: 8px;
+  --token-side-nav-body-list-item-padding-vertical: 4px;
+  --token-side-nav-body-list-item-content-spacing-horizontal: 8px;
+
+  /* Colors: */
+
+  /* SideNav container: */
+  --token-side-nav-foreground-primary: #dedfe3;
+  --token-side-nav-foreground-primary-strong: #fff;
+  --token-side-nav-foreground-faint: #8c909c;
+  --token-side-nav-surface: #0c0c0e;
+
+  // Interactive elements:
+  // base:
+  --token-side-nav-interactive-surface-hover: #3b3d45;
+  --token-side-nav-interactive-surface-active: #656a76;
+
+  // selected / current:
+  --token-side-nav-interactive-selected-surface-default: #656a76;
+  --token-side-nav-interactive-selected-surface-hover: #3b3d45;
+  --token-side-nav-interactive-selected-surface-active: #656a76;
+}
+
+
+// * SideNav Parent (wrapper) component
+// -------------------------------------------------------------------
+
+.hds-side-nav__wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  color: var(--token-side-nav-foreground-primary); // Default color
+  background: var(--token-side-nav-surface);
+}
+
+.hds-side-nav__wrapper-header {
+  padding: var(--token-side-nav-wrapper-padding-vertical) var(--token-side-nav-wrapper-padding-horizontal);
+}
+
+.hds-side-nav__wrapper-body {
+  flex: 1;
+  padding: var(--token-side-nav-wrapper-padding-vertical) var(--token-side-nav-wrapper-padding-horizontal);
+  overflow-y: auto;
+}
+
+.hds-side-nav__wrapper-footer {
+  padding: var(--token-side-nav-wrapper-padding-vertical) var(--token-side-nav-wrapper-padding-horizontal);
+}
+
+// * Header child elements
+// ---------------------------------------
+
+.hds-side-nav-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.hds-side-nav-header__logo {
+  display: block;
+  align-items: center;
+  justify-content: center;
+  width: var(--token-side-nav-header-home-link-logo-width);
+  height: var(--token-side-nav-header-home-link-logo-height);
+}
+
+.hds-side-nav-header__actions {
+  display: flex;
+
+  > * + * { margin-left: var(--token-side-nav-header-actions-spacing); }
+}
+
+.hds-side-nav__home-link {
+  display: block;
+  width: 100%;
+  height: 100%;
+  padding: var(--token-side-nav-header-home-link-padding); // by design
+  // TODO discuss with Cristiano/Christian if this needs to become a component-level token
+  color: var(--token-color-foreground-high-contrast);
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  &:focus,
+  &.mock-focus {
+    @include hds-focus-ring-with-pseudo-element();
+  }
+
+  &:hover,
+  &.mock-hover {
+    background: var(--token-side-nav-interactive-surface-hover);
+  }
+
+  &:active,
+  &.mock-active {
+    background: var(--token-side-nav-interactive-surface-active);
+  }
+}
+
+// class name only, no component:
+.hds-side-nav__dropdown {
```

Comment:
Is there a way to confirm that if this component is on the same page as a regular dropdown, we don't have style conflicts?

Reply:
in the current implementation, the class `hds-side-nav__dropdown` is applied directly to the Dropdown element (as additional class); later once the "dark" styles are finalized, we will use the parent `hds-side-nav-header__actions` container as selector, to avoid the consumers having to manually add the class themselves

in both cases, no risks of style leaking to other dropdowns